client/src/App.js

```diff
@@ -22,6 +23,7 @@ function App() {
 
       {/* Should we be including all this text in header? Should we also be using body? */}
       <header className="App-header">
+        <linearGradient text = "background"/>
```

Comment:
Because our react component doesn't accept any props, we should just leave this as `<linearGradient />`. (also when we do pass props, we don't want any spaces around the `=`)